stile/hsc/base_tasks.py

```diff
@@ -430,24 +450,30 @@ def computeShapes(self, data, calib, do_shape=True, do_err=True, do_psf=True, do
             ixy = numpy.array([mom.getIxy() for mom in moments])
             iyy = numpy.array([mom.getIyy() for mom in moments])
         if do_err:
-            key = data.schema.find("shape.sdss.err").key
-            covariances = numpy.array([src.get(key) for src in data])
-            if sky_coords:
-                cov_ixx = numpy.zeros(covariances[:,0,0].shape)
-                cov_iyy = numpy.zeros(covariances[:,0,0].shape)
-                cov_ixy = numpy.zeros(covariances[:,0,0].shape)
-                for i, (cov, lt) in enumerate(zip(covariances,localLinearTransform)):
-                    cov_ixx[i] = (lt[0,0]**4*cov[0,0] +
-                                  (2.*lt[0,0]*lt[0,1])**2*cov[2,2] + lt[0,1]**4*cov[1,1])
-                    cov_iyy[i] = (lt[1,0]**4*cov[0,0] +
-                                  (2.*lt[1,0]*lt[1,1])**2*cov[2,2] + lt[1,1]**4*cov[1,1])
-                    cov_ixy[i] = ((lt[0,0]*lt[1,0])**2*cov[0,0] +
-                                  (lt[0,0]*lt[1,1]+lt[0,1]*lt[1,0])**2*cov[2,2] +
-                                  (lt[0,1]*lt[1,1])**2*cov[1,1])
+            if 'galaxy' in mask_type and self.config.do_hsm:
+                key = data.schema.find('shape.hsm.regauss.err').key
+                errs = numpy.array([src.get(key) for src in data])
+                sigma_errs = numpy.array(errs.shape)
+                sigma_errs.fill(1.)
```

Comment:
I did not find outputs of moment covariance from `shape.hsm`, so this is the best we can do for now...